model/versioning/VersionedGridFieldDetailForm.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+/**
+ * Extends {@see GridFieldDetailForm}
+ */
+class VersionedGridFieldDetailForm extends Extension {
```

Comment:
What's the rationale for using an extension here if we're always applying it in core anyway?